x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality_details/overview/degraded_fields/degraded_fields.tsx

```diff
@@ -28,8 +30,25 @@ export function DegradedFields() {
   const accordionId = useGeneratedHtmlId({
     prefix: overviewDegradedFieldsSectionTitle,
   });
+  const toggleTextSwitchId = useGeneratedHtmlId({ prefix: 'toggleTextSwitch' });
 
-  const { totalItemCount } = useDegradedFields();
+  const { totalItemCount, toggleCurrentQualityIssues, showCurrentQualityIssues } =
+    useDegradedFields();
+
+  const onChange = (e: { target: { checked: React.SetStateAction<boolean> } }) => {
+    toggleCurrentQualityIssues();
+  };
+
+  const latestBackingIndexToggle = (
+    <EuiSwitch
+      label={overviewDegradedFieldToggleSwitch}
+      checked={showCurrentQualityIssues}
+      onChange={onChange}
```

Comment:
[nit]
```suggestion
      onChange={toggleCurrentQualityIssues}
```

Reply:
good catch, i was testing with local state, then removed the local state logic and missed to remove this function call